BUILDING.md

```diff
@@ -667,6 +670,46 @@ $ ./android-configure /path/to/your/android-ndk
 $ make
 ```
 
+## Installing Node.js on z/OS
+
+### z/OS Prerequisites
+
+Hardware and Software Requirements for installing Node.js on z/OS
+* Hardware
+  * IBM zEnterprise 196 (z196) or newer
+* Software
+  * z/OS Unix System Services enabled
+   * z/OS V2R4 with PTFs:UI64830,UI64837,UI64839,UI64940,UI65567
+   * z/OS V2R3 with PTFs:UI61308,UI61375,UI61747
+  * Integrated Cryptographic Service Facility (ICSF) must be enabled on systems.
+  * Python 3.8.2 or later 3.x versions, required to compile native add-ons.
+    It must also be defined in your PATH environment variable.
+  * Make 4.1 or later, required to compile native add-ons.
+   It must also be defined in your PATH environment variable.
+
+### Installing
```

Comment:
This doesn't belong here as this document is for building Node.js from source. A more appropriate place would be https://github.com/nodejs/nodejs.org/blob/master/locale/en/download/package-manager.md (i.e., https://nodejs.org/en/download/package-manager/) and/or https://github.com/nodejs/nodejs.dev/blob/master/src/pages/docs.tsx (i.e., https://nodejs.dev/docs/).